compiler/damlc/tests/src/DA/Test/IncrementalBuilds.hs

```diff
@@ -149,42 +148,30 @@ tests damlc repl = testGroup "Incremental builds"
               createDirectoryIfMissing True (takeDirectory $ dir </> file)
               writeFileUTF8 (dir </> file) content
           let dar = dir </> "out.dar"
-          callProcessSilent (ShouldSucceed True) damlc ["build", "--project-root", dir, "-o", dar, "--incremental=yes"]
-          callProcessSilent (ShouldSucceed True) repl ["testAll", dar]
+          callProcessSilent damlc ["build", "--project-root", dir, "-o", dar, "--incremental=yes"]
+          callProcessSilent repl ["testAll", dar]
           dalfFiles <- getDalfFiles $ dir </> ".daml/build"
           dalfModTimes <- for dalfFiles $ \f -> do
               modTime <- getModificationTime f
               pure (f, modTime)
           for_ modification $ \(file, content) -> do
               createDirectoryIfMissing True (takeDirectory $ dir </> file)
               writeFileUTF8 (dir </> file) content
-          callProcessSilent (ShouldSucceed True) damlc ["build", "--project-root", dir, "-o", dar, "--incremental=yes"]
+          callProcessSilent damlc ["build", "--project-root", dir, "-o", dar, "--incremental=yes"]
           rebuilds <- forMaybeM dalfModTimes $ \(f, oldModTime) -> do
               newModTime <- getModificationTime f
               pure $ if newModTime == oldModTime
                   then Nothing
                   else Just (makeRelative (dir </> ".daml/build") f -<.> ".daml")
           assertEqual "Expected rebuilds" (Set.fromList $ map normalise expectedRebuilds) (Set.fromList $ map normalise rebuilds)
-          callProcessSilent (ShouldSucceed True) repl ["validate", dar]
-          callProcessSilent shouldSucceed repl ["testAll", dar]
+          callProcessSilent repl ["validate", dar]
+          callProcessSilent' shouldSucceed repl ["testAll", dar]
```

Comment:
I think it would make more sense to just have `callProcessSilentError` and not expose `callProcessSilent'`.

Reply:
Ok. I shall take care of it. Stand by.

Reply:
@cocreature (very exellent),
![done](https://user-images.githubusercontent.com/1500167/78130173-c865ab80-73e6-11ea-835d-4ca607e820ac.jpg)